src/index.html

```diff
@@ -1,12 +1,310 @@
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+
+        </ul>
+
+      </nav>
+
+    </header>
+
+    <main class="main">
+
+      <section class="product-cards">
+        <div class="product-card" data-qa="card">
+          <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="product-card__img"
+        >
+          <div class="product-card__info">
+            <h5 class="product-card__product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h5>
+            <p class="product-card__product-code">Product code: 195434</p>
+            <div class="product-card__review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="product-card__review-amount">Reviews: 5</p>
+            </div>
+            <div class="product-card__price">
+              <p class="product-card__price-text">Price:</p>
+              <p class="product-card__price-value">$2,199</p>
+            </div>
+            <button class="product-card__buy-button" data-qa="hover">Buy</button>
+          </div>
+        </div>
+        <div class="product-card" data-qa="card">
+          <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="product-card__img"
+        >
+          <div class="product-card__info">
+            <h5 class="product-card__product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h5>
+            <p class="product-card__product-code">Product code: 195434</p>
+            <div class="product-card__review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="product-card__review-amount">Reviews: 5</p>
+            </div>
+            <div class="product-card__price">
+              <p class="product-card__price-text">Price:</p>
+              <p class="product-card__price-value">$2,199</p>
+            </div>
+            <button class="product-card__buy-button" data-qa="hover">Buy</button>
+          </div>
+        </div>
+        <div class="product-card" data-qa="card">
+          <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="product-card__img"
+        >
```

Comment:
Need to fix it everywhere
```suggestion
          <img
            src="images/imac.jpeg"
            alt="imac"
            class="product-card__img"
          >
```